src/esm_environment/esm_environment.py

```diff
@@ -86,8 +85,6 @@ def __init__(self, run_or_compile, complete_config=None, model=None):
             for model in complete_config:
                 self.apply_config_changes(run_or_compile, complete_config, model)
 
-        # Add the ENVIRONMENT_SET_BY_ESMTOOLS into the exports
-        self.add_esm_var()
```

Comment:
I have always assumed that some model might have used that. This introduces the following lines in the `.run` file:

```
export ENVIRONMENT_SET_BY_ESMTOOLS=TRUE
```

Does anyone know if that's the case? @pgierz @denizural

Reply:
ECHAM via auto-tools uses that. Does the `add_esm_var` do **only** the job of setting the variable? If yes, then I would propose to leave it in.

Reply:
To be more specific: PaleoDyn's ECHAM via Autotools uses that...The Geomar ECHAM might be different.

Reply:
I have not seem anyone using `ENVIRONMENT_SET_BY_ESMTOOLS`. It is still nice as a debug feature though.